tests/src/Context/AssetDistributionContext.php

```diff
@@ -295,7 +295,7 @@ public function cleanTestingEntities(): void {
       if (!empty($entities)) {
         foreach ($entities as $entity) {
           $entity->skip_notification = TRUE;
-          $entity->delete();
+          @$entity->delete();
```

Comment:
Was this causing trouble?

Reply:
Yes, when for some reasons the entity is deleted in other process.